sources/editor/Stride.Assets.Presentation/Templates/Assets/Scripts/Physics/DebugShapesRender.cs

```diff
@@ -23,43 +23,37 @@ public override async Task Execute()
             return;
 
         var shapesRenderState = new RenderStage("PhysicsDebugShapes", "Main");
-            compositor.RenderStages.Add(shapesRenderState);
-            var meshRenderFeature = compositor.RenderFeatures.OfType<MeshRenderFeature>().First();
-            meshRenderFeature.RenderStageSelectors.Add(new SimpleGroupToRenderStageSelector
-            {
-                EffectName = "StrideForwardShadingEffect",
-                RenderGroup = (RenderGroupMask)(1 << (int)RenderGroup),
-                RenderStage = shapesRenderState,
-            });
-            meshRenderFeature.PipelineProcessors.Add(new WireframePipelineProcessor { RenderStage = shapesRenderState });
-            debugRenderer.DebugRenderStages.Add(shapesRenderState);
+        compositor.RenderStages.Add(shapesRenderState);
+        var meshRenderFeature = compositor.RenderFeatures.OfType<MeshRenderFeature>().First();
+        meshRenderFeature.RenderStageSelectors.Add(new SimpleGroupToRenderStageSelector
+        {
+            EffectName = "StrideForwardShadingEffect",
+            RenderGroup = (RenderGroupMask)(1 << (int)RenderGroup),
+            RenderStage = shapesRenderState,
+        });
+        meshRenderFeature.PipelineProcessors.Add(new WireframePipelineProcessor { RenderStage = shapesRenderState });
+        debugRenderer.DebugRenderStages.Add(shapesRenderState);
 
-            var simulation = this.GetSimulation();
-            if (simulation != null)
-                simulation.ColliderShapesRenderGroup = RenderGroup;
+        var simulation = this.GetSimulation();
+        if (simulation != null)
+            simulation.ColliderShapesRenderGroup = RenderGroup;
 
-            var enabled = false;
-            while (Game.IsRunning)
+        var enabled = false;
+        while (Game.IsRunning)
+        {
+            if (Input.IsKeyDown(Keys.LeftShift) && Input.IsKeyDown(Keys.LeftCtrl) && Input.IsKeyReleased(Keys.P))
             {
-                if (Input.IsKeyDown(Keys.LeftShift) && Input.IsKeyDown(Keys.LeftCtrl) && Input.IsKeyReleased(Keys.P))
+                if (simulation != null)
                 {
                     if (simulation != null)
```

Comment:
Double `if` with the same condition. Also, not very important, but isn't it better now to `is not null`?